src/components/admin/article/components/ImageUrlField.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ValidatedHttpsUrlField } from 'components/admin/form-components/validated-fields';
+
+const ImageUrlField = (props) => {
+  const onChange = (event) => {
+    props.updateImage(event.target.value);
+  };
+
+  return (
+    <ValidatedHttpsUrlField
+      name="image"
+      value={props.image}
+      floatingLabelText="Image (Remember to use https:// not http://)"
+      disabled={props.disabled}
+      onChange={onChange}
+      fullWidth
+    />
+  );
+};
+
+ImageUrlField.propTypes = {
+  image: PropTypes.string,
```

Comment:
Should this not be required?

Reply:
Hmmm, my logic when writing it was that it technically isn't required, but it would be an odd UX if it wasn't defined for sure. 

Let me think about this for a bit.

Reply:
Yeah, I feel like I would rather get the propType error than notice that my field is a non-interactive div and then figure out that it was because I forgot to give the prop.

Also now I'm looking at it, this should be called imageUrl to be consistent with the rest of the renamings we did